packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -308,6 +312,22 @@ - (void)setMyLocationButtonEnabled:(BOOL)enabled {
   _mapView.settings.myLocationButton = enabled;
 }
 
+- (BOOL)setMapStyle:(NSString*)mapStyle {
+  if (mapStyle == (id)[NSNull null] || mapStyle.length == 0) {
```

Comment:
nit: we don't seem to cast to `(id)` elsewhere, do we need it?

Reply:
I get an IDE warning about comparison between incompatible pointer types, we could either cast it to `(NSString*)` or `(id)` to address that and I chose the latter. It shouldn't be a big deal if we choose to remove the cast but seems reasonable to leave it.

Reply:
Totally reasonable was just wondering why this is different from the other similar checks in this file. Taking another look I guess it's because the other instances here are access `NSArray` elements.